docs/release-notes/Percona-Everest-1.5.0-(2025-02-07).md

```diff
@@ -0,0 +1,154 @@
+# What's new in Percona Everest 1.5.0
+
+!!! warning
+    Google Container Registry (GCR) is scheduled to be deprecated and will officially shut down on March 18, 2025. All versions of Percona Everest prior to 1.4.0 depend on images hosted on GCR, meaning that downloading those images will fail after the shutdown date. We strongly recommend upgrading to Percona Everest version 1.4.0 as soon as possible. If you do not upgrade, Percona Everest will no longer function.
+
+    For more details, refer to the [Container Registry Deprecation documentation](https://cloud.google.com/artifact-registry/docs/transition/prepare-gcr-shutdown){:target="_blank"}.
+
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quick-install.md).
+
+
+??? info "Release summary at a glance"
+
+    |**Sr. No**|**Release summary**|**Description**|
+    |---------|---------------------|---------|
+    | **1.**|[RBAC: Integration with IdP groups](#streamlining-role-based-access-control-with-idp-group-integration)|Assign RBAC policies to user groups obtained from an external IdP|
+    | **2.**|[Operators support](https://docs.percona.com/everest/release-notes/Percona-Everest-1.4.0-%282025-01-07%29.html#__tabbed_1_4)|Support for PXC operator 1.16.1|
+    | **3.**|[New features](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#new-features)|Check out the new features introduced in Percona Everest 1.5.0|
+    | **4.**|[Improvements](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#improvements)|Discover all the enhancements featured in Percona Everest 1.5.0|
+    | **5.**|[Bugs](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#bugs)|Find out about all the bugs fixed in Percona Everest 1.5.0|
+    | **6.**|[Known limitations](https://docs.percona.com/everest/release-notes/Percona-Everest-1.3.0-%282024-11-18%29.html#known-limitations)|Discover all the known limitations in Percona Everest 1.5.0|
+
+
+## Release highlights
```

Comment:
Big highlight missing: RBAC GA :tada: :rocket:

Reply:
Ok. Will add it